### ISSUE TITLE
optional argument -M

### ISSUE THREAD

**Issue by tobiv92** [davidemms/OrthoFinder#71]
when I try to calculate the genes tree using FastTree with the -M argument
`orthofinder -f ExampleDataset_fasttree -M msa`
I get following error:
```
Unrecognised argument: -M
ERROR: An error occurred, please review previous error messages for more information.
```

**Comment by davidemms** [comment]
Hi

Have you managed to resolve this? I can't think of any reason the parsing of the command line options wouldn't work. Are you using the latest version of OrthoFinder, or at least a version that supports that option? Running "orthofinder -h" will tell you the version number and will also let you check that it supports the "-M" option. 

All the best
David

**Comment by tobiv92** [comment]
Hey there,
I run it in a computer cluster, it has v1.1.2 installed. the option `-M` is not showing up in the help...

**Comment by davidemms** [comment]
Hi

Ok, that'll be it. That option is available from version 1.1.4: https://github.com/davidemms/OrthoFinder/releases/tag/1.1.4

All the best
David
